.github/workflows/publish_docker.yml

```diff
@@ -0,0 +1,41 @@
+name: publish docker
```

Comment:
do we need a new workflow? another job within the original publish workflow should be enough I think.

Reply:
Yea you are probably right. I think I separated just for testing. I'll look to combine them